antora.yml

```diff
@@ -27,7 +27,7 @@ asciidoc:
     # for the project
     context: docs
     ibmsg: IBM Style Guide
-    ibmsg-url: https://www.pearson.com/us/higher-education/program/De-Respinis-IBM-Style-Guide-The-Conventions-for-Writers-and-Editors/PGM270594.html?tab=contents
+    ibmsg-url: https://www.ibm.com/docs/en/ibm-style
```

Comment:
Meaning we replace a public resource by a private one? I don't have access to this page.

Reply:
Updated, leaving the original links, and also adding the new ones.